.github/pull_request_template.md

```diff
@@ -1,3 +1,5 @@
+Please add a changelog description here. This will help populate the repo changelog and docs site release notes.
```

Comment:
```suggestion
Please add a one-paragraph summary here, suitable for a release notes description. This will help with documentation.
```

Reply:
Technically this won't end up in the change log, just the docs website release notes.